optimize-distro/assembly-distro-production.xml

```diff
@@ -20,7 +20,6 @@
       <directory>src/config/</directory>
       <excludes>
         <exclude>${exclude.from.config}</exclude>
-        <exclude>environment-config-demo.yaml</exclude>
       </excludes>
       <outputDirectory>config/</outputDirectory>
```

Comment:
the `curl.exe` file is also only used by the demo files, so please remove it as well